src/components/common/applicantAccordions/ApplicantAccordion/ApplicantAccordion.style.ts

```diff
@@ -0,0 +1,102 @@
+import { css, Theme } from '@emotion/react';
+import { flexGenerator } from '@styles/generator';
+
+export const applicantAccordionLayout = css`
+  ${flexGenerator('column', 'center', 'center')};
+  gap: 2rem;
+`;
+
+/* 체크 및 신청자 정보 <--space-between--> 아코디언 버튼  */
+export const applicantLayoutStyle = css`
+  ${flexGenerator('row', 'space-between', 'center')}
```

Comment:
```suggestion
  ${flexGenerator('row', 'space-between')};
```

이렇게 사용하실 수도 있습니다! 수정안하셔도돼요

Reply:
수정할래요